README.md

```diff
@@ -20,17 +20,11 @@
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+### Install - npm
 
-### Setup
+### Usage - git clone git@github.com:Addisu87/webpack-demo.git
 
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+### Run tests - npm run build then npm start
```

Comment:
- [ ] [OPTIONAL] It would be nicer if only the title has a bold text, I would advice adding more content to your getting started section to make it look better and professional.